src/qubx/backtester/account.py

```diff
@@ -154,7 +154,9 @@ def _process_new_data(self, instrument: Instrument, data: Quote | OrderBook | Tr
             return
         for r in ome.process_market_data(data):
             if r.exec is not None:
-                self.order_to_instrument.pop(r.order.id)
+                # TODO: why do we pop here if it is popped later after send
+                if r.order.id in self.order_to_instrument:
+                    self.order_to_instrument.pop(r.order.id)
```

Comment:
theoretically someone can try to try cancel order in on_execution callback (it'd be called directly in send call) - so it's removed here